src/hybrid_reader.h

```diff
@@ -31,13 +31,14 @@ typedef struct {
   heap_t *topResults;              // Sorted by score (max heap).
   //heap_t *orderedResults;        // Sorted by id (min heap) - for future use.
   size_t numIterations;
+  bool ignoreScores;               // Ignore the document scores, only vector score matters.
 } HybridIterator;
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-IndexIterator *NewHybridVectorIterator(VecSimIndex *index, char *score_field, KNNVectorQuery query, VecSimQueryParams qParams, IndexIterator *child_it);
+IndexIterator *NewHybridVectorIterator(VecSimIndex *index, char *score_field, KNNVectorQuery query, VecSimQueryParams qParams, IndexIterator *child_it, bool ignoreScores);
```

Comment:
once https://github.com/RediSearch/RediSearch/pull/2647 will be merged, this constructor signature will be unreadable. Please create a struct with everything we need for initializing the hybrid iterator